Libraries/src/Amazon.Lambda.PowerShellHost/PowerShellFunctionHost.cs

```diff
@@ -149,6 +149,7 @@ private IAsyncResult BeginInvoke(string input, ILambdaContext context)
             this._ps.Commands?.Clear();
             this._ps.Streams.Verbose?.Clear();
             this._ps.Streams.Error?.Clear();
+            this._ps.Streams.Debug?.Clear();
```

Comment:
Looks like we are missing clearing `Information` and `Warning` here. Can we add those?

Reply:
Added